freqtrade/optimize/backtesting.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self.config['exchange']['uid'] = ''
         self.config['dry_run'] = True
         self.exchange = Exchange(self.config)
+        self.fee = self.exchange.get_fee()
```

Comment:
can you add a test for this? 

probably here:
https://github.com/freqtrade/freqtrade/blob/c413e94f83d0a12378e18725f81aa325569148d5/freqtrade/tests/optimize/test_backtesting.py#L296-L307